localstack-core/localstack/services/cloudwatch/provider_v2.py

```diff
@@ -105,6 +106,11 @@
 AWS_MAX_DATAPOINTS_ACCEPTED: int = 1440
 
 
+@cloudwatch_feature.Alarm
+def delete_alarms_helper():
+    pass
+
+
```

Comment:
just added this to verify that we can also decorate functions, and that the data will be collected as well.